packages/solana/lib/src/rpc/dto/token_amount.dart

```diff
@@ -24,4 +25,7 @@ class TokenAmount {
 
   /// Token amount as a string, accounting for decimals.
   final String? uiAmountString;
+
+  /// Token amount as a decimal number.
+  final double uiAmount;
```

Comment:
AFAIK, `uiAmount` is deprecated, that's why we didn't include it.

Reply:
Great spot. Thanks. Adding [official doc](https://docs.solana.com/developing/clients/jsonrpc-api#token-balances-structure) as clarification.

Reply:
In that case, let's rather remove it from the DTO, unless there's some strong reason to keep it.

Reply:
Removed.